build/release.mk

```diff
@@ -65,7 +65,7 @@ endif
 
 	@# Download prepackaged plugins
 	@for plugin_package in $(PLUGIN_PACKAGES) ; do \
-		curl -s https://api.github.com/repos/mattermost/$$plugin_package/releases/latest | grep browser_download_url | cut -d '"' -f 4 | wget -qi - -P  $(DIST_PATH)/prepackaged_plugins/ ;\
```

Comment:
Besides the switch from `curl` to `wget`, can you comment on the other changes here? Are they an artifact of parsing the `wget` vs. `curl` output?

Reply:
The `-i` is a GNU extension, that isn't available on busybox's wget which is used on Alpine.

Reply:
I'm not against using `curl`, or `wget`, but using both isn't needed.

Reply:
Note also:
- `wget` is usually installed by default, `curl` not, on Debian, [Ubuntu](http://releases.ubuntu.com/18.04/ubuntu-18.04.2-desktop-amd64.manifest) and CentOS operating systems (not Docker)
- with `curl` we need to take the basename from the URL to save a file, because the `-o` defines the destination filename.

Reply:
Agree to avoid using both -- I'd propose converging on `curl` for now unless @crspeller has opinions otherwise.

Reply:
I'm ok to go with curl :+1: